libraries/AP_Camera/AP_Camera_Params.cpp

```diff
@@ -81,6 +81,12 @@ const AP_Param::GroupInfo AP_Camera_Params::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_OPTIONS", 10, AP_Camera_Params, options, 0),
 
+    // @Param: _MNT_INST
+    // @DisplayName: Mount instance
```

Comment:
Let's rename this to "Camera Mount instance".  It's good to have the subsystem name at the front when possible.